org.eclipse.buildship.kotlin/build.gradle

```diff
@@ -0,0 +1,21 @@
+buildscript {
+    ext.kotlin_version = '1.0.4'
+    repositories.jcenter()
+    dependencies.classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
+}
+
+apply plugin: eclipsebuild.BundlePlugin
+apply plugin: 'kotlin'
+
+dependencies {
+    compile 'org.gradle:gradle-core:3.3'
+    compile 'org.gradle:gradle-process-services:3.3'
```

Comment:
Some random class? Can you be more specific? As far as I can see this should only be using the TAPI, not Gradle core.